### PR TITLE
Fix submodules defining  ansible-kubectl

### PR DESCRIPTION
**Pull request by lucendio**

### CODE REVIEW

**Comment by lucendio** [comment]
@arianvp what about `ansible-helm`

**Comment by arianvp** [comment]
@lucendio yes ansible-helm is also unused as far as I know

**Review by arianvp — APPROVED** [review]

File: ansible/Makefile [hunk 1]
```diff
@@ -43,7 +43,7 @@ ifndef LOG_SINCE
 endif
 	ansible-playbook ${ANSIBLE_DIR}/get-logs.yml \
 		-i ${ENV_DIR}/gen/terraform-inventory.yml \
-		-i ${ENV_DIR}/inventory.yml \
```
Comment (arianvp) [inline ansible/Makefile:46]: Just checking. is the dropping of `.yml` intentsional herE?

Comment (lucendio) [reply ansible/Makefile:46]: the whole change was not meant to be in here. Will remove it.

**Pull request merged**